tutorials/using-edge/production-deploy-to-ase-gpu.ipynb

```diff
@@ -0,0 +1,458 @@
+   "outputs": [],
+   "source": [
+    "# Set the web service configuration (using default here)\n",
+    "from azureml.core.model import InferenceConfig\n",
+    "\n",
+    "# from azureml.core.webservice import AksWebservice\n",
+    "from azureml.core.conda_dependencies import CondaDependencies\n",
+    "from azureml.core.environment import Environment, DEFAULT_GPU_IMAGE\n",
+    "\n",
+    "env = Environment(\"deploytoedgeenv\")\n",
+    "# Please see [Azure ML Containers repository](https://github.com/Azure/AzureML-Containers#featured-tags)\n",
+    "# for open-sourced GPU base images.\n",
+    "env.docker.base_image = DEFAULT_GPU_IMAGE\n",
+    "env.python.conda_dependencies = CondaDependencies.create(\n",
+    "    conda_packages=[\"tensorflow-gpu==1.12.0\", \"numpy\"],\n",
+    "    pip_packages=[\"azureml-contrib-services\", \"azureml-defaults\"],\n",
+    ")\n",
+    "\n",
+    "inference_config = InferenceConfig(entry_script=\"score.py\", environment=env)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create container image in Azure ML\n",
+    "Use Azure ML to create the container image. This step will likely take a few minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# provide name of azure contaienr image and tag\n",
+    "imagename = \"tfgpu\"\n",
+    "imagelabel = \"0.2\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Builds an image in ACR.\n",
+    "\n",
+    "package = Model.package(\n",
+    "    ws,\n",
+    "    [model],\n",
+    "    inference_config=inference_config,\n",
+    "    image_name=imagename,\n",
+    "    image_label=imagelabel,\n",
+    ")\n",
+    "package.wait_for_creation(show_output=True)\n",
+    "\n",
+    "print(\"ACR:\", package.get_container_registry)\n",
+    "print(\"Image:\", package.location)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup Azure Stack Edge \n",
+    "\n",
+    "Follow [documentation](https://review.docs.microsoft.com/en-us/azure/databox-online/azure-stack-edge-gpu-deploy-sample-module-marketplace?branch=release-preview-ase-gpu) to setup compute and validate if GPU on ASE are up and runing."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup Azure IoT Edge device\n",
+    "\n",
+    "Follow [documentation](https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux) to setup a Linux VM as an Azure IoT Edge device"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploy container to Azure IoT Edge device"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.image import ContainerImage\n",
```

Comment:
this is deprecated, please do not import anything from `azureml.core.image`

Reply:
removed